maven/blade.servicebuilder.test/src/main/java/com/liferay/blade/samples/servicebuilder/test/AddTestData.java

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
Let me try moving it to liferay-workspace/tests. that would solve the other concerns in gradle and maven